src/pages/MetricDocs/Sections/MetricName.js

```diff
@@ -146,7 +146,7 @@ const Metrics: ComponentType<*> = ({ metrics, setUri }) => {
         </div>
         <RenderMetrics data={ data }
                        filterFunc={ () => true }
-                       download={ URLs["genericApiMetricSearch"] + `?search=${userInput}` }
+                       download={ URLs["genericApiMetricSearch"] + `?search=${userInput}` + `&category=${category}` + `&tags=${types.join(",")}` }
```

Comment:
We need to use `utils.createQuery` to generate this one. This means that we should predefine the parameters array (possibly in a `useEffect`) and ensure that empty params are excluded.